data/data-pipeline/data_pipeline/etl/sources/cdc_svi_index/etl.py

```diff
@@ -0,0 +1,112 @@
+import pandas as pd
+import numpy as np
+
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.utils import get_module_logger
+from data_pipeline.score import field_names
+
+logger = get_module_logger(__name__)
+
+
+class CDCSVIIndex(ExtractTransformLoad):
+    """CDC SVI Index class ingests 2018 dataset located
+    here: https://www.atsdr.cdc.gov/placeandhealth/svi/index.html
+    Please see the README in this module for further details.
+    """
+
+    def __init__(self):
+        self.OUTPUT_PATH = self.DATA_PATH / "dataset" / "cdc_svi_index"
+
+        self.CDC_SVI_INDEX_URL = "https://svi.cdc.gov/Documents/Data/2018_SVI_Data/CSV/SVI2018_US.csv"
```

Comment:
🎉

Reply:
Could you ask Jorge to delete the SVI file in s3?

Reply:
Deleted @lucasmbrown-usds @saran-ahluwalia